src/CMakeLists.txt

```diff
@@ -332,6 +332,11 @@ set(libcommon_files
   ${mds_files})
 set_source_files_properties(ceph_ver.c
   APPEND PROPERTY OBJECT_DEPENDS ${CMAKE_BINARY_DIR}/src/include/ceph_ver.h)
+
+if(WITH_EVENTTRACE)
+  message(STATUS " Using EventTrace class.")
```

Comment:
might want to drop this feedback as it's not an option/feature enabled/identified by us. it's enabled by user.